keras_nlp/models/bert/bert_presets.py

```diff
@@ -45,13 +49,15 @@
             "max_sequence_length": 512,
             "num_segments": 2,
         },
-        "vocabulary": "uncased_en",
         "description": (
             "Small size of BERT where all input is lowercased. "
             "Trained on English Wikipedia + BooksCorpus."
         ),
         "weights_url": "https://storage.googleapis.com/keras-nlp/models/bert_small_uncased_en/model.h5",
         "weights_hash": "08632c9479b034f342ba2c2b7afba5f7",
+        "vocabulary_url": "https://storage.googleapis.com/keras-nlp/models/bert_base_uncased_en/vocab.txt",
+        "vocabulary_hash": "64800d5d8528ce344256daf115d4965e",
+        "lowercase": True,
```

Comment:
Perhaps there should be a separate tokenizer config field in the dict, to store things like "lowercase"?

Reply:
I like a config dict for this overall as well. I think it is a more general purpose and allows to grow a bit better. 

Even for sentencepiece, which is a "vocabulary" file format that stores a lot of it's own metadata, I see a need for this coming up. If you look at say the [t5 config](https://huggingface.co/t5-base/blob/main/config.json) in huggingface, they have a lot of preprocessing params that flatten straight in to the base config, including special token ID numbers, sequence lengths, even tasks specific prefixes.

I think collocating preprocessing config params in a dict gives us more room to grow.